spec/breadcrumbs/breadcrumb_spec.rb

```diff
@@ -0,0 +1,47 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+require 'bugsnag/breadcrumbs/breadcrumb'
+
+describe Bugsnag::Breadcrumbs::Breadcrumb do
+  describe "initialize" do
```

Comment:
I prefer `describe "#initialize"` (with the `#`) so that the failure message is `Bugsnag::Breadcrumbs::Breadcrumb#initialize should assign arguments correctly` (similarly for the `#to_h` describe block)